app/Console/Kernel.php

```diff
@@ -64,5 +66,6 @@ protected function schedule(Schedule $schedule)
         $schedule->command('tables:optimise')->daily();
         $schedule->command('msl:generate')->hourlyAt([25, 55]);
         $schedule->command('networkdata:update')->everyMinute()->withoutOverlapping();
+        $schedule->command('stands:assign-arrival')->everyTwoMinutes();
```

Comment:
What does performance on this command look like?

Reply:
On my Homestead VM, runs in < 5 seconds after a fresh network data update.